client/src/featureform/register.py

```diff
@@ -1590,7 +1591,97 @@ class Customer:
             properties=properties,
         )
 
+### Problems: the user is forced to specify th entity column. If they put the same entity name as the class that they are defining the multiple features in, will it error? 
+### If the mention a different class name, then will it get applied and work?
+class MultiFeatureColumnResource(ColumnResource):
+    def __init__(
+        self,
+        dataset: SourceVariant,
+        df: pd.DataFrame,
+        entity_column: Union[Entity, str] = "",
+        variant: str = "",
+        owner: str = "",
+        inference_store: Union[str, OnlineProvider, FileStoreProvider] = "",
+        timestamp_column: str = "",
+        include_columns: List[str] = [],
```

Comment:
It's bad practice to use a default empty list for inputs since the state is shared between all creations of this object. It won't have much of an impact but it's better to start getting rid of them, the better thing to do is to default to None here and if it's none we create an empty list

Reply:
Python yuck. You can also use the `or` keyword for shorter assignment.
self.include_columns = include_columns or []

Reply:
yup